### DISCUSSION TITLE
First public release

### DISCUSSION THREAD

**Discussion by ScratMan** [ScratMan/HASmartThermostat#1] [Announcements]
Some code updates to the original aendle smart thermostat integration.

**Full Changelog**: https://github.com/ScratMan/HASmartThermostat/compare/0.5.1...2021.11.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ScratMan/HASmartThermostat/releases/tag/2021.11.1'>First public release</a>.</em>

**Comment by bigbread59** [comment]
Great job.
A beer for you !!!
